src/core/friendly_errors/sketch_verifier.js

```diff
@@ -0,0 +1,118 @@
+import * as espree from 'espree';
+
+/**
+ * @for p5
+ * @requires core
+ */
+function sketchVerifier(p5, fn) {
+  /**
+   * Fetches the contents of a script element in the user's sketch.
+   * 
+   * @method fetchScript
+   * @param {HTMLScriptElement} script
+   * @returns {Promise<string>}
+   */
+  fn.fetchScript = async function (script) {
+    if (script.src) {
+      const contents = await fetch(script.src).then((res) => res.text());
```

Comment:
There's a chance this request will fail depending on access control policies of the server, we might want to wrap lines 16/17 in a `try`/`catch`. If there's an error, maybe we can just log the error and return an empty string?

Reply:
Yeah CORS is a potential problem here. I'm thinking would it be possible to catch the error, print a friendly message about it, then degrade into a lower version of the script content grabbing eg. the `'' + window.setup` technique used in the current sketch reader?

That way there is still some checks going on while advise on enabling CORS on the server will still be available.

Reply:
Added a TODO as a future item!